docs/architecture/security/principles/01-servers-are-not-trusted.mdx

```diff
@@ -0,0 +1,37 @@
+# P01 - Servers are not trusted
+
+Clients do not need to trust Bitwarden server or web infrastructure to be good actors in supplying
+data to keep an account's encrypted content secure. In other words, the special status of providing
+a network sync for client data does not enable a server, or any entity between the server and
+client, to take any action that would aid in decryption of a user's vault by the server.
+
+This is what we mean when we sometimes refer to "End-to-end encrypted."
+
+## Account key sharing as a feature
+
+This principle does not mean that clear text data are never shared, but rather that any such
```

Comment:
```suggestion
This principle does not mean that clear text data is never shared, but rather that any such
```

Reply:
unfortunately data is plural so I think this is already correct

Reply:
Can you double check this? https://dictionary.cambridge.org/dictionary/english/data says that it is an Uncountable singular noun; for American English, the singular verb is preferred:

> [U] 	Uncountable or singular noun: a noun that has no plural.

> [+ sing/pl verb] 	A noun that refers to a group of people acting collectively. When used in the singular it can be followed by either a singular or a plural verb in British English. In American English a singular verb is preferred.

Reply:
looks like it might be an audience or me being out of date thing. I had grown up learning singular is _datum_, but it looks like dictionaries are switching.

I still find lots of references claiming either is true, but I'm happy capitulating here and contributing to English as a living language!